src/signature-request/clause.ts

```diff
@@ -0,0 +1,24 @@
+import * as t from "io-ts";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+
+export const ClauseTitle = NonEmptyString;
```

Comment:
i'm wondering if we should take those types from the generated ones from the API specs.